Homework2/Task1/Classes/MatrixTracker.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Task1.Classes
+{
+ class MatrixTracker<T>
+ {
+  // is it okey to store event information like this?
+  private T _storedEventElement;
```

Comment:
As this class emulates Matrix client and there were no requirements for it, it is OK to store info at desired manner.